ircd/send.c

```diff
@@ -1199,6 +1198,38 @@ sendto_monitor(struct Client *source_p, struct monitor *monptr, const char *patt
 	msgbuf_cache_free(&msgbuf_cache);
 }
 
+/* sendto_monitor_with_capability_butserial()
+ *
+ * inputs	- monitor nick to send to, caps, negate caps, whether to send to clients having current serial, format, va_args
+ * outputs	- message to local users monitoring the given nick
+ * side effects -
+ */
+void
+sendto_monitor_with_capability_butserial(struct Client *source_p, struct monitor *monptr, int caps, int negcaps, bool skipserial, const char *pattern, ...)
+{
+	va_list args;
+
+	va_start(args, pattern);
+	_sendto_monitor_with_capability_butserial(source_p, monptr, caps, negcaps, skipserial, pattern, &args);
+	va_end(args);
+}
+
+/* sendto_monitor()
+ *
+ * inputs	- monitor nick to send to, format, va_args
+ * outputs	- message to local users monitoring the given nick
+ * side effects -
+ */
+void
+sendto_monitor(struct Client *source_p, struct monitor *monptr, const char *pattern, ...)
+{
+	va_list args;
+
+	va_start(args, pattern);
+	_sendto_monitor_with_capability_butserial(source_p, monptr, NOCAPS, NOCAPS, false, pattern, &args);
```

Comment:
Eliminate current_serial abuse

```suggestion
	_sendto_monitor_with_capability(source_p, monptr, NOCAPS, NOCAPS, pattern, &args);
```